src/cljam/algo/vcf_indexer.clj

```diff
@@ -1,21 +1,22 @@
 (ns cljam.algo.vcf-indexer
-  "Indexer of VCF."
+  "Indexer of VCF/BCF."
   (:require [cljam.io.csi :as csi]
+            [cljam.io.util :as io-util]
             [cljam.io.vcf :as vcf])
   (:import cljam.io.vcf.reader.VCFReader))
```

Comment:
Unused

Reply:
I use "io-util/vcf-reader?" to get variant-file-type.

Reply:
So, it’s okay to remove the line..?

Reply:
Sorry. I misunderstood that io-util is unnecessary.